### ISSUE TITLE
How can I bind the component?

### ISSUE THREAD

**Issue by atdhehk** [sabakugaara/angular-pell#3]
Hello Gaara, can you please help me how to bind the component?

**Comment by MickL** [comment]
Could you explain what you mean with "bind the component"? Did you mean this: https://github.com/sabakugaara/angular-pell/issues/4 ? If yes, I have the same question.

**Comment by atdhehk** [comment]
Yeah sure, I mean when component got loaded I want to load a content (html
string) than modifying that content. Let’s say we have a post and I want to
edit and if I have a button to switch to editing mode, I want the content
of my post to load into editor
